src/command_line_parser.cc

```diff
@@ -1861,53 +1934,26 @@ CLParser::VerifyOptions()
     Usage("Cannot specify --async and --sync simultaneously.");
   }
 
-  if (params_->using_concurrency_range && params_->using_old_options) {
-    Usage("Cannot use deprecated options with --concurrency-range.");
-  } else if (params_->using_old_options) {
-    if (params_->dynamic_concurrency_mode) {
-      params_->concurrency_range.end = params_->max_concurrency;
-    }
-    params_->concurrency_range.start = params_->concurrent_request_count;
-  }
-
-  if (params_->using_request_rate_range && params_->using_old_options) {
-    Usage("Cannot use concurrency options with --request-rate-range.");
-  }
-
-  if (!params_->schedule.empty() && params_->warmup_request_count > 0) {
-    Usage("Cannot use warmup options with --schedule");
-  }
-
-  if (!params_->schedule.empty() && params_->using_concurrency_range) {
-    Usage("Cannot use concurrency options with --schedule");
```

Comment:
Getting rid of checks specific to deprecated options since they leverage other existing checks now, same for the inference load mode checks here--they leverage other existing checks now.